docs/design/datacontracts/GCCover.md

```diff
@@ -0,0 +1,29 @@
+# Contract GCCover
+
+This contract encapsulates support for [GCCover](../coreclr/jit/investigate-stress.md) (GC stress testing) in the runtime.
+
+## APIs of contract
+
+```csharp
+public virtual TargetPointer? GetGCCoverageInfo(NativeCodeVersionHandle codeVersionHandle);
+```
+
+## Version 1
+
+Data descriptors used:
+| Data Descriptor Name | Field | Meaning |
+| --- | --- | --- |
+| NativeCodeVersionNode | GCCoverageInfo | pointer to GC coverage info |
+
+Contracts used:
+| Contract Name |
+| --- |
+| CodeVersions |
+| RuntimeTypeSystem |
+
+### Getting GCCoverageInfo for a NativeCodeVersion
+```csharp
+public virtual TargetPointer? GetGCCoverageInfo(NativeCodeVersionHandle codeVersionHandle);
```

Comment:
```suggestion
public virtual bool TryGetGCCoverageInfo(NativeCodeVersionHandle codeVersionHandle, [NotNullWhen(true)] out TargetPointer? pointer);
```

Reply:
Changed to `public virtual TargetPointer GetGCCoverageInfo(NativeCodeVersionHandle codeVersionHandle);` as discussed above.